ui/src/containers/SubOrgForm/index.js

```diff
@@ -477,6 +485,46 @@ class SubOrgForm extends Component {
               onDismiss={this.onExpirationDismiss}
               readonly={!this.props.isSiteAdmin} />
           </div>
+
```

Comment:
Think we can add some inline help to these forms, akin to what we did in this PR: https://github.com/LearningLocker/learninglocker/pull/1194#issuecomment-400658189